VirtualLibrary/Localization/Translations.cs

```diff
@@ -1013,14 +1035,19 @@ public static class Translations
 
         public static string GetTranslatedString(string id)
         {
-            var lang = Registration.GetUserLanguageSetting();
+            var lang = Opening.GetUserLanguageSetting();
             if (lang == null)
-                lang = "EN";
-            string translatedString;
-            var foundTranslation = translations.Find(Translation => Translation.id == id);
-            var pair = foundTranslation.translatedStrings.Find(LanguageValuePair => LanguageValuePair.language == lang);
-            translatedString = pair.value;
-            return translatedString;
+                lang = "LT";
+            //string translatedString;
+            //var foundTranslation = translations.Find(Translation => Translation.id == id);
+            // var pair = foundTranslation.translatedStrings.Find(LanguageValuePair => LanguageValuePair.language == lang);
+            //translatedString = pair.value
+            Dictionary<string, string> tempDictionary = newTranslations[lang];
+            if (tempDictionary.ContainsKey(id) == true)
+            {
+                return tempDictionary[id];
+            }
+            return "WORST STRING";
```

Comment:
Good job! I'm only wondering what happens when "worst string" is returned?

Reply:
oh, I left this one because I was planning to check all the translations after the button is fixed. This return is basically for checking if there are strings left that haven't been translated yet. Once I will check if everything is working properly, I will probably just delete this or rename it to something that makes more sense like "String has no translation"

Reply:
Oh okey, thanks